addons/website/static/src/components/views/page_search_model.js

```diff
@@ -0,0 +1,64 @@
+/** @odoo-module */
+
+import { useService } from "@web/core/utils/hooks";
+
+import { Domain } from '@web/core/domain';
+import { SearchModel } from '@web/search/search_model';
+
+const { onWillStart, useState } = owl;
+
+
+export class PageSearchModel extends SearchModel {
+    setup() {
```

Comment:
Not sure what's the convention for OWL classes, are we not adding the default documentation markers anymore? In this case, I am reading the file without knowing what you are overriding or creating (I surely can deduce it but...). Also for some I would really like some comment to understand what's going on.

Reply:
Doesn't seems to be a thing in new owl in web (from 474 occurences, only 27 are not in legacy).
I checked some other apps extending web things and there is not much of that either.

But anyway, I added it, we can stick to our internal guidelines regardless?
+ Added quick comments.

For tracking purpose: lunch is doing the kind of same code, see `search_model.js` in lunch

Reply:
> But anyway, I added it, we can stick to our internal guidelines regardless?

I think it is worth it indeed :shrug: Also serves as a visual marker between function TBH.

If you can add a blank line before the first `@param`... thanks :pray:

Reply:
Grr, I am getting old, I should not have forgot about that line, changed, thanks